lib/PuppeteerSharp/Helpers/CancellationTokenSourceHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading;
+
+namespace PuppeteerSharp.Helpers
+{
+    /// <summary>
+    /// CancellationTokenSource Helper.
+    /// </summary>
+    public static class CancellationTokenSourceHelper
+    {
+        /// <summary>
+        /// Creates a CancellationToken with a timeout.
+        /// </summary>
+        /// <param name="timeout">Timeout in milliseconds.</param>
+        /// <returns>The timed cancellation token.</returns>
+        public static CancellationToken GetCancellationToken(int timeout)
```

Comment:
Maybe rename to something like `GetTimingOutCancellationToken` ? Or rename the class to `TimingOutCancellationToken` and then name the method just `Get`